src/bundle/ui-dev/src/modules/universal-discovery/components/content-create-widget/content.create.widget.js

```diff
@@ -115,9 +116,11 @@ const ContentCreateWidget = () => {
                     <div className="ibexa-extra-actions__section-header">{selectLanguageLabel}</div>
                     <div class="ibexa-extra-actions__section-content">
                         <Dropdown
+                            dropdownListRef={DropdownPortalRefContext}
```

Comment:
Is it correct? In Dropdown component as I see you want to get ref element (from useRef hook) and here you pass context